testgrid/config/config.yaml

```diff
@@ -1933,6 +1933,92 @@ dashboards:
   - name: kubelet-non-cri-1.6
     test_group_name: ci-kubernetes-node-kubelet-non-cri-1.6
 
+# Blocking tests for the next release
+- name: release-master-blocking
```

Comment:
We want to know what broken tests will be blocking the next release, long before we actually cut the branch for that release.